plover/engine.py

```diff
@@ -321,6 +321,7 @@ def _on_machine_state_changed(self, machine_state):
 
     def _consume_engine_command(self, command):
         # The first commands can be used whether plover has output enabled or not.
+        original_command = command
         command = command.lower()
         if command == 'resume':
```

Comment:
If the above suggestion is accepted, command_name would be used instead of "command"